2_Operations/investment-strategy.md

```diff
@@ -0,0 +1,30 @@
+# Investment Strategy
+
+Niteo keeps reserves to [cover close to 12 months of average expenses](/5_People/profit-sharing.md). To not lose this money to inflation, we invest it into appreciating assets.
+
+
+## Short-Term Pool
+
+Reserves in safe and liquid assets. We try to keep the currency equally split between EUR and USD. Around 30% of total reserves.
```

Comment:
```suggestion
Reserves in safe and liquid assets. We try to keep the currency equally split between EUR and USD to not have all our eggs in one basket. Minimum 30% of all reserves should be in this pool so that we have cash on hand if we need to act quick.
```